src/scripts/items/PokemonItem.ts

```diff
@@ -42,4 +42,5 @@ ItemList['Combee']               = new PokemonItem(GameConstants.PokemonItemType
 ItemList['Burmy (plant)']        = new PokemonItem(GameConstants.PokemonItemType['Burmy (plant)'], 5000);
 ItemList['Cherubi']              = new PokemonItem(GameConstants.PokemonItemType.Cherubi, 5000);
 ItemList['Spiritomb']            = new PokemonItem(GameConstants.PokemonItemType.Spiritomb, 5000);
+ItemList['Zorua']                = new PokemonItem(GameConstants.PokemonItemType.Zorua, 5000);
```

Comment:
I think we should probably bump the price up a bit as this is pretty far into the game.
Meloetta below is 50k

Reply:
Meloetta is a legendary. And also an alternate form, therefor optional for progress to Kalos at some point in the future, whenever that may be. So it can have an outrageous price. Zorua tho, very much required. Not quite the same situation.